src/app/auth/pages/login/login.component.scss

```
@@ -0,0 +1,18 @@
+.panel-login {
+background-color: rgb(60, 73, 84, 0.8);
+border-radius: 15px;
+width: 30%;
+height: 40%;
+}
+.login {
+background-image: url(fondo-prelogin-ejc.jpg);
```

Guardar la imagen en la carpeta assets y llamarla desde alla

```suggestion
background-image: url(../../../assets/fondo-prelogin-ejc.jpg);
```